### PR TITLE
feat(variables): use new env variable endpoints with react query

### PR DESCRIPTION
**Pull request by ctjhoa**
# What does this PR do?

https://qovery.atlassian.net/browse/FRT-883

### CODE REVIEW

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery  preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery  preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> You are going to thank you later if it contains tests

**Review by RemiBonnet — APPROVED** [review]
Thanks, nice to see one less with Redux 🎉

**Comment by RemiBonnet** [comment]
Just two things:

- The loader for the buttons on the modals doesn't works
- And, it missing the animations after creating a variable : https://www.loom.com/share/f2b38875c0184ca89386a71423465130?sid=e36a9838-5b9e-41ad-8cfc-58a570aaa8ce)

File: libs/domains/variables/data-access/src/lib/domains-variables-data-access.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    const response = await variableMainCallsApi.importEnvironmentVariables(
+      serviceId,
+      serviceType,
+      variableImportRequest
+    )
+    return response.data
+  },
+}
+
+export type VariablesKeys = inferQueryKeys<typeof variables>
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply libs/domains/variables/data-access/src/lib/domains-variables-data-access.ts]: No, a bad habit of mine
https://github.com/Qovery/console/pull/970/commits/58e4f2952ce42d16c2b38c7a47205c3a08918f6d

**Review by RemiBonnet — APPROVED** [review]

**Pull request merged**